src/main/java/com/hollingsworth/arsnouveau/client/gui/radial_menu/GuiRadialMenu.java

```diff
@@ -246,13 +246,11 @@ public boolean keyPressed(int key, int scanCode, int modifiers) {
         return super.keyPressed(key, scanCode, modifiers);
     }
 
-    @Override
-    public boolean mouseClicked(double p_mouseClicked_1_, double p_mouseClicked_3_, int p_mouseClicked_5_) {
```

Comment:
this breaks the hotkey functionality and mod compat that may mixin or listen to events called by mouseClicked(...)

```java
    @Override
    public boolean keyPressed(int key, int scanCode, int modifiers) {
        int adjustedKey = key - 48;
        if (adjustedKey >= 0 && adjustedKey < radialMenuSlots.size()) {
            selectedItem = adjustedKey == 0 ? radialMenuSlots.size() : adjustedKey;
            selectedItem = selectedItem - 1; // Offset by 1 because 0 based indexing but users see 1 indexed
            mouseClicked(0, 0, 0);
            return true;
        }
        return super.keyPressed(key, scanCode, modifiers);
    }
```